README.md

```diff
@@ -49,34 +48,22 @@ Once the mutation succeeds, we could request apollo to send a refetch request to
 Optimistic update allows us to write predictable data into cache if we know ahead what it will be. This allows us to provide quicker experience feedback for our users. Eg. in an address book when the user marks an address as favourite, we know the mutation will respond with true but the mutation might take a few hundred ms to respond. So we use optimistic response to mark the address as favourite and show quicker UI feedback to our users. When the optimistic response and the actual mutation response match, apollo will ignore the response. If the actual response is different from optimistic response apollo will go through usual failure flow.
 
 - use optimistic response only for edits where you can predict the response well
-- don't use optimistic response for creates since you will need an id to insert into cache that you only usually get in mutation response
+- don't use optimistic response for creates since you will need an id to insert into cache that you only usually get in mutation response. You can work around this with extra work, by either generating ids in the FE (if possible) - or by optimistically adding new entities with temporary ids, and then removing those entities once the mutation returns.
```

Comment:
Thought I'd throw out a few options for how you might be able to still optimistically respond to creates in case it's necessary.

For my use case in fieldbook, I've found that generating id's in the FE works fine.